views/js/review/provider/qtiTestReviewProvider.js

```diff
@@ -139,6 +140,28 @@ define([
 
             this.assetManager = assetManagerFactory();
 
+            // set tabindex=0 in navigation links and hide other links from navigation
+            $(document).find("li a:not([tabindex])").attr("tabindex", 0);
+            $(document).find("a:not([tabindex])").attr("tabindex", -1);
```

Comment:
Please avoid to target the whole page, prefer scope instead to avoid side effects. Remember the component might be integrated in any place. So it should not interfere with the surrounding.

Reply:
removed